src/services/OfferService.js

```diff
@@ -24,7 +24,7 @@ export default class OfferService {
   ): Promise<string> {
     const token = createToken()
 
-    const form = new forms[city](formData)
+    const form = new forms[city]['Schema'](formData)
```

Comment:
What is 'Schema' here? Looks like a magic constant.

Reply:
It is somehow one, each forms[city] object must have a mongoose schema in order to insert something into the db.